README.md

```diff
@@ -39,3 +39,63 @@ You might want to:
 - Read abut [developing components](https://github.com/canonical-web-and-design/react-components/blob/master/HACKING.md).
 - Find out how to [publish to NPM](https://github.com/canonical-web-and-design/react-components/blob/master/PUBLISH-NPM-PACKAGE.md).
 - Know how to [publish the docs](https://github.com/canonical-web-and-design/react-components/blob/master/PUBLISHING-DOCS.md) to GitHub Pages.
+
+## Developing locally using this repository
+
+You may wish to link this library directly to your projects while developing locally.
+
+You can do this by cloning this repo to your local workspace;
+
+```shell
+git clone https://github.com/canonical-web-and-design/react-components
+```
+
+If you then drop into that folder and run;
+
+```shell
+yarn link
```

Comment:
There's also a script to help with this: https://github.com/canonical-web-and-design/react-components/blob/master/package.json#L99 it also links react like you do in the next section.

There's an unlink-package script to reverse it: https://github.com/canonical-web-and-design/react-components/blob/master/package.json#L109

Reply:
Thanks - didn't spot these..have now added them to the updated README